# Falling Number Workshop 2019  
Pacific Northwest Quality Council | Portland, OR  

[![FN_talk_slides](https://github.com/shantel-martinez/Lab_Resources/blob/master/example_img/SMartinez%20FN%20Workshop%2001.30.2019.png)](link_to_Slides)

[Slide deck](Link_to_Slide_Deck) | Slide deck as [PDF](Link_To_PDF) 

## PNW GWAS Publication  
The presentation above has been [published](https://www.frontiersin.org/articles/10.3389/fpls.2018.00141/full) by Frontiers in Plant Science an open-sourced article.   
Martinez SA, Godoy J, Huang M, et al (2018) Genome-wide Association Mapping for Tolerance to Preharvest Sprouting and Low Falling Numbers in Wheat. Front Plant Sci 9:1â€“16. [doi: 10.3389/fpls.2018.00141](https://www.frontiersin.org/articles/10.3389/fpls.2018.00141/full)     

## Resources  
The Falling Numbers [Website](http://steberlab.org/project7599.php)  

**PHS/Dormancy Genes and RefSeqv1.0 Positions**   

| Gene   |  | Chrm | Pos_Start    | Pos_End      | RefSeqv1.0_Gene_Name     | Reference        | Gene_Seq            |
| ------- | ------------ | ---- | ------------ | ------------ | ----------------------------------------- | ---------------------- | ------------------------------------------------------------ |
| TaSdr Sdr  | TaSdr-A1     | 2A   | 158,452,164  | 158,453,676  | TraesCS2A01G191400                        | Zhang et al., 2014     | [KF021988](http://getentry.ddbj.nig.ac.jp/getentry/na/KF021988?filetype=html) |
|   | TaSdr-B1     | 2B   | 200,574,062  | 200,572,560  | TraesCS2B02G215300                        |                        | [KF021990](http://getentry.ddbj.nig.ac.jp/getentry/na/KF021990?filetype=html) |
|   | TaSdr-D1     | 2D   | 142,668,463  | 142,666,958  | TraesCS2D01G196200                        |                        | [KF021992](http://getentry.ddbj.nig.ac.jp/getentry/na/KF021992?filetype=html) |
| Tamyb10   R  | R-A1         | 3A   | 703,906,112  | 703,904,221  | - | Himi et al., 2011      | [AB599721](https://www.ebi.ac.uk/ena/data/view/AB599721)               |
|   | R-B1         | 3B   | 757,917,663  | 757,920,399  | TraesCS3B01G515900                        |                        | [AB191459](https://www.ebi.ac.uk/ena/data/view/AB191459)                 |
|    | R-D1         | 3D   | 570,799,694  | 570,803,366  | TraesCS3D01G468400                        |                        | [AB191460](https://www.ebi.ac.uk/ena/data/view/AB191460)                 |
| TaVp-1 Vp1      | TaVP-1A      | 3A   | 659,556,434  | 659,552,099  | TraesCS3A01G417300                        | Yang et al., 2014      | [Vp-1A](https://www.ncbi.nlm.nih.gov/gene/?term=Vp-1A) |
|   | TaVP-1B      | 3B   | 693,342,691  | 693,338,300  | TraesCS3B01G452200                        |                        | [AJ400713](http://getentry.ddbj.nig.ac.jp/getentry/na/AJ400713?filetype=html) |
|    | TaVP-1D      | 3D   | 525,473,897  | 525,469,502  | TraesCS3D01G412800                        |                        | [AJ400714](http://getentry.ddbj.nig.ac.jp/getentry/na/AJ400714?filetype=html) |
| TaPHS1 TaMFT    | TaMFT-3A     | 3A   | 7,294,362    | 7,297,912    | TraesCS3A02G006600                        | Nakamura et al., 2011  | [G1UE17 ](https://www.uniprot.org/uniprot/G1UE17)                       |
| TaDOG1       | TaDOG1L4     | 3A   | 67,123,800   | 67,124,332   | TraesCS3A01G103500                        | Ashikawa et al., 2013 | [AB555729](https://www.ebi.ac.uk/ena/data/view/AB555729)                 |
|   |              | 3B   | 91,147,446   | 91,147,986   | TraesCS3B01G120900                        |                        |                                                              |
|   |              | 3D   | 58,107,736   | 58,108,268   | TraesCS3D01G105800                        |                        |                                                              |
| TaMKK3  Phs-1    | TaMKK3-A     | 4A   | 605,019,000 **a** | 605,023,000 | TraesCSU01G167000                         | Torada et al., 2016    | [LC091368](http://getentry.ddbj.nig.ac.jp/getentry/na/LC091368?filetype=html) |
|   | TaMKK3-B     | 5B   | 710,226,100  | 710,230,860  | TraesCS5B01G565100                        |                        |                                                              |
|   | TaMKK3-D     | 5D   | 556,557,389  | 556,553,063  | TraesCS5D01G549600                        |                        | [LC091372](http://getentry.ddbj.nig.ac.jp/getentry/na/LC091372?filetype=html) |
| TaQsd1  Qsd1 **b**    | TaA          | 5A   | 432,446,370  | 432,446,370  | TraesCS5D01G224200                        | Onishi et al., 2017    | [Qsd1](https://static-content.springer.com/esm/art%3A10.1186%2Fs12864-017-3880-6/MediaObjects/12864_2017_3880_MOESM2_ESM.zip) |
|    | TaB          | 5B   | 387,744,682  | 387,740,080  | TraesCS5B01G214700                        |                        |                                                              |
|   | TaD          | 5D   | 332,043,225  | 332,043,225  | TraesCS5D01G224200                        |                        |                                                              |  

**a** RefSeqv1.0 maps TaMKK3-A to the unknown chrm, however based on previous research, TaMKK3-A should be between TraesCS4A01G314700 and TraesCS4A01G314800   
**b** the ortholog of the barley dormancy gene HvQsd1. The Onishi et al. 2017 study compared sequence of dormant and non-dormant lines, however further work needs to be published to see if TaQsd1 genes effect seed dormancy

## Research Funding   
PNW GWAS Research: 
> WA Grain Commission Projects: 7599, 5389, and 126593   
> USDA NIFA Projects: 2015-05798, and 2016-68004-24770   
> USDA ARS Project: 424575

Cornell University Research: 
> USDA NIFA Project: 2017-07109

FN Workshop:  
>  Project:   
