# Falling Number Workshop 2019  
Pacific Northwest Quality Council | Portland, OR  

[![FN_talk_slides](https://github.com/shantel-martinez/Lab_Resources/blob/master/example_img/SMartinez%20FN%20Workshop%2001.30.2019.png)](https://github.com/shantel-martinez/FNWorkshop2019/blob/master/SMartinez%20FN%20Workshop%2001.30.2019.pdf)

Download [PDF](https://github.com/shantel-martinez/FNWorkshop2019/raw/master/SMartinez%20FN%20Workshop%2001.30.2019.pdf) | PDF [with notes](https://github.com/shantel-martinez/FNWorkshop2019/raw/master/SMartinez%20FN%20Workshop%2001.30.2019%20Notes.pdf) 

## PNW PHS GWAS Publication  
The majority of the presentation above has been [published](https://www.frontiersin.org/articles/10.3389/fpls.2018.00141/full) by Frontiers in Plant Science as an open-sourced article.   
> Martinez SA, Godoy J, Huang M, et al (2018) Genome-wide Association Mapping for Tolerance to Preharvest Sprouting and Low Falling Numbers in Wheat. Front Plant Sci 9:1â€“16. doi: 10.3389/fpls.2018.00141 

## Top PHS QTL 
The QTL identified in Martinez et al., 2018 that were significant in more than one environment for spike-wetting tests.   

| QTL_VisibleSprout | Index    | Name                        | Chr   | Pos_Start   | Pos_End     | Environment                | Favorable Allele | Primer_1                  | Primer_2                  | Primer_C                 |
| -------------- | -------- | --------------------------- | ----- | ----------- | ----------- | -------------------------- | ---------------- | ------------------------- | ------------------------- | ------------------------ |
| QPHS.wsu-1A.2  | IWB6759  | BS00021780_51               | 1A    | 591,950,221 | 591,950,321 | P14 d3                     | **A**/G              | acacgacACcaagtgcacA       | acacgacACcaagtgcacG       | ccttcagttgtgagctGtgaG    |
|                | IWB77968 | wsnp_Ex_c3887_7051325       | 1A    | -           | -           | P14 d3                     | **A**/G              | gActCgcCataacTtttgttTcA   | gActCgcCataacTtttgttTcG   | acagagatggaccaacctgG     |
| QPHS.wsu-1B.2  | IWB64868 | RFL_Contig5022_2001         | 1B    | 663,047,736 | 663,047,836 | C14 d4                     | **A**/G              | tgctgctgatcGcgctgT        | tgctgctgatcGcgctgC        | cgtttcacacagttcgcg       |
|                | IWB31676 | GENE-0115_375               | 1A    | 574,831,161 | 574,831,061 | P16 d3                     | **A**/G              | gggtatgggtacagctttataaacA | gggtatgggtacagctttataaacC | agtgcttcataccaccttgC     |
| QPHS.wsu-1D*   | IWB71680 | Tdurum_contig46389_2038     | 3B | 29,355,667  | 29,355,567  | P14, P16 d6, P16 SI        | **A**/G              | agctgcatAtgtttccaaAtacT   | agctgcatAtgtttccaaAtacC   | tcaAaaggCttcttCAaaaaCCAA |
| QPHS.wsu-2D    | IWB7652  | BS00027456_51               | 2D    | 351,983,138 | 351,983,038 | C14 d4, C14 d5, C14 d6, d7 | T/**C**              | -                         | -                         | -                        |
|                | IWA8544  | wsnp_RFL_Contig3960_4401914 | 2D    | 277,937,915 | 277,937,815 | P16 d3, P16 d4, C14 SI     | **A**/G              | ggaatatagatgggaccactagcA  | ggaatatagatgggaccactagcG  | acagataaaaggaaccggcA     |
| QPHS.wsu-5A.1  | IWB10998 | BS00078242_51               | 5A    | 469,612,740 | 469,612,640 | C14 SI                     | T/**C**              | acaccaattgcccttatctaccT   | acaccaattgcccttatctaccC   | aattcttgagggcgtccgag     |
| QPHS.wsu-5A.2  | IWB60303 | RAC875_c716_198             | 5D | 424,663,024 | 424,663,123 | C15 d7                     | A/**G**              | agtctgactggaacatcccA      | agtctgactggaacatcccG      | ggaagttcgagacgtacatattcA |
|                | IWB10250 | BS00068178_51               | 5A    | 540,052,088 | 540,052,188 | P15 d5                     | **T**/C              | catcagtccaagaagacgatgaT   | catcagtccaagaagacgatgaC   | ggtgtctccgtcccaagtac     |
| QPHS.wsu-6B    | IWB76583 | wsnp_Ex_c1398_2676484       | 6B    | 417,238,843 | 417,238,653 | P14 d6, P14 SI, P14 d5     | **A**/G              | tggtttccatgtgaattAgagtcA  | tggtttccatgtgaattAgagtcG  | tgggaaaacaaaggggcgG      |
|                | IWB57747 | RAC875_c41604_1001          | 6B    | 309,775,161 | 309,775,061 | P16 SI                     | **A**/G              | ctgcttcgccacaccacA        | ctgcttcgccacaccacG        | gatggcgaagaacaacaggc     |
| QPHS.wsu-6D*   | IWB49280 | Kukri_rep_c107605_164       | 6D    | -           | -           | P15 d7                     | **A**/G              | GgctgaatggTtgaagtttgataaT | GgctgaatggTtgaagtttgataaC | caaagcatgaaagaaagggagT   |
| QPHS.wsu-7B.2* | IWB7099  | BS00022463_51               | 7B    | 608,810,414 | 608,810,514 | C14 d5, C14 SI, P16 SI     | A/**G**              | gtcagacatgaaagtagttgactcA | gtcagacatgaaagtagttgactcG | tgtaGccagtttcctttgtctcC  |

Chr, Pos_Start, and Pos_End are locations of the Illumina 90k SNP probe alignment to the RefSeqv1.0 alignment. Note that some probes may aligned to more than one chromosome, however reported above are the loactions with the highest identity.  
Favorable alleles for decreased visible sprouting are shown in BOLD  
The Illumina 90k SNP array sequences were ran against the PolyMarker software to create [KASP markers](http://polymarker.tgac.ac.uk/Markdown?md=DesignedPrimers) for the majority of the array. KASP Marker sequences are shown bove indicated by Primer_1, Primer_2, and Primer_C.       

## Resources  
**The Falling Numbers** [**Website**](http://steberlab.org/project7599.php)  

**PHS/Dormancy Genes and RefSeqv1.0 Positions**    
A number of genes in wheat have been cloned and characterized for PHS tolerance or grain dormancy. Since the wheat reference genome is now [publically available](https://www.wheatgenome.org/News/Latest-news/Reference-Sequence), we can [BLAST](https://urgi.versailles.inra.fr/blast_iwgsc/blast.php) the PHS gene sequences (below) against the RefSeqv1.0 alignment and identify the positions on the chromosome. Below are the estimated start and end positions based on the gene sequences available by the authors, in the publication, or public databases. The positions and links to the gene sequences may be useful to cross-reference with future PHS QTL or GWAS analyses.  

| Gene   |  | Chrm | Pos_Start    | Pos_End      | RefSeqv1.0_Gene_Name     | Reference        | Gene_Seq            |
| ------- | ------------ | ---- | ------------ | ------------ | ----------------------------------------- | ---------------------- | ------------------------------------------------------------ |
| TaSdr /    Sdr  | TaSdr-A1     | 2A   | 158,452,164  | 158,453,676  | TraesCS2A01G191400                        | [Zhang et al., 2014](https://link.springer.com/article/10.1007/s11032-013-9935-8)     | [KF021988](http://getentry.ddbj.nig.ac.jp/getentry/na/KF021988?filetype=html) |
|   | TaSdr-B1     | 2B   | 200,574,062  | 200,572,560  | TraesCS2B02G215300                        |                        | [KF021990](http://getentry.ddbj.nig.ac.jp/getentry/na/KF021990?filetype=html) |
|   | TaSdr-D1     | 2D   | 142,668,463  | 142,666,958  | TraesCS2D01G196200                        |                        | [KF021992](http://getentry.ddbj.nig.ac.jp/getentry/na/KF021992?filetype=html) |
| Tamyb10 /  R  | R-A1         | 3A   | 703,906,112  | 703,904,221  | - | [Himi et al., 2011]()      | [AB599721](https://www.ebi.ac.uk/ena/data/view/AB599721)               |
|   | R-B1         | 3B   | 757,917,663  | 757,920,399  | TraesCS3B01G515900                        |                        | [AB191459](https://www.ebi.ac.uk/ena/data/view/AB191459)                 |
|    | R-D1         | 3D   | 570,799,694  | 570,803,366  | TraesCS3D01G468400                        |                        | [AB191460](https://www.ebi.ac.uk/ena/data/view/AB191460)                 |
| TaVp-1 / Vp1      | TaVP-1A      | 3A   | 659,556,434  | 659,552,099  | TraesCS3A01G417300                        | [Yang et al., 2014](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5097096/)      | [Vp-1A](https://www.ncbi.nlm.nih.gov/gene/?term=Vp-1A) |
|   | TaVP-1B      | 3B   | 693,342,691  | 693,338,300  | TraesCS3B01G452200                        |                        | [AJ400713](http://getentry.ddbj.nig.ac.jp/getentry/na/AJ400713?filetype=html) |
|    | TaVP-1D      | 3D   | 525,473,897  | 525,469,502  | TraesCS3D01G412800                        |                        | [AJ400714](http://getentry.ddbj.nig.ac.jp/getentry/na/AJ400714?filetype=html) |
| TaPHS1 / TaMFT    | TaMFT-3A     | 3A   | 7,294,362    | 7,297,912    | TraesCS3A02G006600                        | [Nakamura et al., 2011](http://www.plantcell.org/content/23/9/3215)  | [G1UE17 ](https://www.uniprot.org/uniprot/G1UE17)                       |
| TaDOG1       | TaDOG1L4     | 3A   | 67,123,800   | 67,124,332   | TraesCS3A01G103500                        | [Ashikawa et al., 2013](http://www.sciencedirect.com/science/article/pii/S0168945213000678) | [AB555729](https://www.ebi.ac.uk/ena/data/view/AB555729)                 |
|   |              | 3B   | 91,147,446   | 91,147,986   | TraesCS3B01G120900                        |                        |                                                              |
|   |              | 3D   | 58,107,736   | 58,108,268   | TraesCS3D01G105800                        |                        |                                                              |
| TaMKK3 / Phs-1    | TaMKK3-A     | 4A   | 605,019,000 **a** | 605,023,000 | TraesCSU01G167000                         | [Torada et al., 2016](http://www.cell.com/current-biology/fulltext/S0960-9822(16)30010-0?_returnURL=http%3A%2F%2Flinkinghub.elsevier.com%2Fretrieve%2Fpii%2FS0960982216300100%3Fshowall%3Dtrue)    | [LC091368](http://getentry.ddbj.nig.ac.jp/getentry/na/LC091368?filetype=html) |
|   | TaMKK3-B     | 5B   | 710,226,100  | 710,230,860  | TraesCS5B01G565100                        |                        |                                                              |
|   | TaMKK3-D     | 5D   | 556,557,389  | 556,553,063  | TraesCS5D01G549600                        |                        | [LC091372](http://getentry.ddbj.nig.ac.jp/getentry/na/LC091372?filetype=html) |
| TaQsd1 / Qsd1 **b**    | TaA          | 5A   | 432,446,370  | 432,446,370  | TraesCS5D01G224200                        | [Onishi et al., 2017](https://doi.org/10.1186/s12864-017-3880-6)    | [Qsd1](https://static-content.springer.com/esm/art%3A10.1186%2Fs12864-017-3880-6/MediaObjects/12864_2017_3880_MOESM2_ESM.zip) |
|    | TaB          | 5B   | 387,744,682  | 387,740,080  | TraesCS5B01G214700                        |                        |                                                              |
|   | TaD          | 5D   | 332,043,225  | 332,043,225  | TraesCS5D01G224200                        |                        |                                                              |  

**a** RefSeqv1.0 maps TaMKK3-A to the unknown chrm, however based on previous research, TaMKK3-A should be between TraesCS4A01G314700 and TraesCS4A01G314800   
**b** the ortholog of the barley dormancy gene HvQsd1. The Onishi et al. 2017 study compared sequence of dormant and non-dormant lines, however further work needs to be published to see if TaQsd1 genes affect seed dormancy

## Research Funding   
Washington State University Research: 
> WA Grain Commission Projects: 7599, 5389, and 126593   
> USDA NIFA Projects: 2015-05798, and 2016-68004-24770   
> USDA ARS Project: 424575

Cornell University Research: 
> USDA NIFA Project: 2017-07109

